Makefile

```diff
@@ -331,6 +331,7 @@ test-e2e: ## Build and run e2e tests
 # Think of it as a Context Manager that:
 # - Before test-time, brings up a `make up`
 # - After test-time, tears it all down and dumps artifacts.
+.SILENT:
```

Comment:
You need to add the name of the target you want to silence as a prerequisite, as with `.PHONY`. As written, I believe this silences _everything_.